events/message.js

```diff
@@ -0,0 +1,23 @@
+module.exports = async (client, message) => {
```

Comment:
Are you doing async here just for performance reasons? Otherwise, I don't think it's needed. You don't do any awaits here.

Reply:
You're right 👍